LLMonFHIR/FHIR Display/FHIRResourcesView.swift

```diff
@@ -51,9 +59,39 @@ struct FHIRResourcesView: View {
                 .sheet(isPresented: $showSettings) {
                     SettingsView()
                 }
+                .sheet(isPresented: $showMultipleResourcesChat) {
+                    resourceChatView
+                }
+                .task {
+                    allResourcesArray = await fhirStandard.resources
+                    await interpretMultipleResources()
+                }
```

Comment:
You can move this functionality to the closure in the `OnboardingActionsView`. We can also set the `allResourcesArray` in there as well.